docs/source/commands/trtllm-serve/run-benchmark-with-trtllm-serve.md

```diff
@@ -44,7 +50,7 @@ TensorRT LLM distributes the pre-built container on [NGC Catalog](https://catalo
 You can launch the container using the following command:
 
 ```bash
-docker run --rm -it --ipc host -p 8000:8000 --gpus all --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/tensorrt-llm/release:x.y.z
+docker run --rm -it --ipc host -p 8000:8000 --gpus all --ulimit memlock=-1 --ulimit stack=67108864 nvcr.io/nvidia/tensorrt-llm/release:1.2.0rc6
```

Comment:
x.y.z was replaced by 1.2.0rc6 inadvertently by the docs building workflow. will have to revert this.